2019-06-04-post-template.md

```diff
@@ -0,0 +1,44 @@
+---
+slug: "post-template"
+title: Wonderful title
+package_version: 0.1.0
+authors:
+  - Author Name
+date: 2019-06-04
+categories: blog
+topicid:
+tags:
+- Software Peer Review
+- R
+- community
+# delete the line below
+# if you have no preferred image
+# for Twitter cards
+twitterImg: img/blog-images/2019-06-04-post-template/name-of-image.png
+---
+
+Everywhere in this template (YAML, paths to images), you should change "post-template" to the slug of your post, and "2019-06-04" to the publication date.
+
+Introduction including outline of the post.
+
+### First awesome section
+
+I like Hugo[^1]. Yep this was a footnote.
```

Comment:
```suggestion
I like Hugo[^1]. Yes, that is how you add a footnote.
```

Reply:
I changed the words in case "yep" (which I would use :-) ) is too colloquial for people who are less used to English